assetmantle/assetlist.json

```diff
@@ -0,0 +1,26 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_id": "mantle-1",
+    "assets": [
+      {
+        "description": "The native token of Asset Mantle",
+        "denom_units": [
+          {
+            "denom": "umntl",
+            "exponent": 0
+          },
+          {
+            "denom": "mntl",
+            "exponent": 6
+          }
+        ],
+        "base": "umntl",
+        "name": "AssetMantle",
+        "display": "mntl",
+        "symbol": "MNTL",
+        "logo_URIs": {
+          "png": "https://avatars.githubusercontent.com/u/97742402?s=200&v=4"
```

Comment:
Could we pull this into this repo.  (make a subfolder called images inside the assetmantle folder)

Reply:
Going to merge for now, please do this in a new PR